plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -126,6 +128,27 @@ class read_only {
    };
    get_info_results get_info(const get_info_params&) const;
 
+   struct get_transaction_status_params {
+      string                       id;
+   };
+
+   struct get_transaction_status_results {
+      string                               state;
+      std::optional<uint32_t>              block_number;
+      std::optional<chain::block_id_type>  block_id;
+      std::optional<fc::time_point>        block_timestamp;
+      std::optional<fc::time_point>        expiration;
+      uint32_t                             head_number;
+      chain::block_id_type                 head_id;
+      fc::time_point                       head_timestamp;
+      uint32_t                             irreversible_number;
```

Comment:
Init `uint32_t`s

Reply:
Addressed in 16e0fd3848e7d23f79599796d0550b96d3cf6086